Makefile

```diff
@@ -63,11 +66,33 @@ test-unit:
 	$(GO) test $(GOFLAGS) -run $(TESTS) ./cmd/... ./pkg/... $(TESTFLAGS) -v
 
 test-e2e:
-	operator-sdk test local ./test/e2e --namespaced-manifest deploy/kubernetes-manifests.yaml --namespace istio-system
+	KUBECONFIG=$(KIND_KUBECONFIG) \
+		operator-sdk test local ./test/e2e --namespaced-manifest deploy/operator-and-rbac.yaml --namespace istio-system
 
 .PHONY: bootstrap
 bootstrap:
 ifndef HAS_DEP
 	go get -u github.com/golang/dep/cmd/dep
 endif
 	dep ensure
+
+create-kindcluster:
+ifndef HAS_KIND
+	@echo "installing kind"
+	GO111MODULE="on" go get sigs.k8s.io/kind@v0.5.1
+endif
+	kind create cluster --name local-kind  --image kindest/node:v1.15.3
```

Comment:
should we add `export KIND_KUBECONFIG="$(kind get kubeconfig-path --name="local-kind")"` here? `KIND_KUBECONFIG` was not set for me in the next step.

Reply:
@michelleN when we need to set the KUBECONFIG this command has to be used:
```
KIND_KUBECONFIG = $(shell kind get kubeconfig-path --name=local-kind)
```
as shown here:
```
install-smi-crds:
	kubectl apply -f https://raw.githubusercontent.com/deislabs/smi-adapter-istio/master/deploy/crds/crds.yaml \
		--kubeconfig=$(KIND_KUBECONFIG)
```